File: app/src/main/java/com/metahumanz/pacilread/stats/annual/AnnualReportStyle.java
package com.metahumanz.pacilread.stats.annual;

public enum AnnualReportStyle {
    QUIET("quiet", "静读留白"),
    HIGHLIGHT("highlight", "高光节奏");

    public final String slug;
    public final String label;

    AnnualReportStyle(String slug, String label) {
        this.slug = slug;
        this.label = label;
    }
}

File: app/src/main/kotlin/com/metahumanz/pacilread/stats/annual/AnnualReportStyle.kt
package com.metahumanz.pacilread.stats.annual

enum class AnnualReportStyle(
    @JvmField val slug: String,
    @JvmField val label: String,
) {
    QUIET("quiet", "静读留白"),
    HIGHLIGHT("highlight", "高光节奏"),
}
